### DISCUSSION TITLE
[Tree] Tree component with draggable

### DISCUSSION THREAD

**Discussion by mmailaender** [skeletonlabs/skeleton#2173] [Ideas]
We have a use case where we need to reorder a tree component. For that, it would help us a lot if the [Tree Component](https://www.skeleton.dev/components/tree-views) would offer a draggable functionality like [Atlassian](https://atlaskit.atlassian.com/packages/confluence/tree/example/drag_and_drop_with_nesting) or [Ant](https://ant.design/components/tree#components-tree-demo-draggable) offers in React.

**Comment by endigo9740** [comment]
Moved to a feature request issue here: https://github.com/skeletonlabs/skeleton/issues/2175
